controller/api/views.py

```diff
@@ -2,176 +2,117 @@
 RESTful view classes for presenting Deis API objects.
 """
 
-from __future__ import absolute_import
-from __future__ import unicode_literals
-
 from django.conf import settings
-from django.contrib.auth.models import AnonymousUser, User
 from django.core.exceptions import ValidationError
-from django.utils import timezone
-from guardian.shortcuts import assign_perm
```

Comment:
I try to be [PEP8 compliant](https://www.python.org/dev/peps/pep-0008/#imports) by putting each import on a separate line, although `flake8` doesn't enforce this.